testsuite/config/__init__.py

```diff
@@ -50,8 +50,14 @@ def __init__(self, name, default, **kwargs) -> None:
             messages={"condition": "{value} is not valid exposer"},
         ),
         Validator("control_plane.managedzone", must_exist=True, ne=None),
-        Validator("control_plane.clusterissuer", must_exist=True, ne=None),
-        Validator("letsencrypt.clusterissuer", must_exist=True, ne=None),
+        (
+            Validator("control_plane.issuer.name", must_exist=True, ne=None)
+            & Validator("control_plane.issuer.kind", must_exist=True, ne=None, is_in={"Issuer", "ClusterIssuer"})
+        ),
+        (
+            Validator("letsencrypt.issuer.name", must_exist=True, ne=None)
+            & Validator("letsencrypt.issuer.kind", must_exist=True, ne=None, is_in={"Issuer", "ClusterIssuer"})
```

Comment:
`ne` is probably not needed, since you use `is_in`

Reply:
Addressed